sdk/storage/azblob/container/client.go

```diff
@@ -421,3 +421,12 @@ func (c *Client) SubmitBatch(ctx context.Context, bb *BatchBuilder, options *Sub
 		Version:     resp.Version,
 	}, nil
 }
+
+// FilterBlobs operation finds all blobs in the container whose tags match a given search expression.
+// https://docs.microsoft.com/en-us/rest/api/storageservices/find-blobs-by-tags
```

Comment:
The URL needs to be corrected.
```suggestion
// https://docs.microsoft.com/en-us/rest/api/storageservices/find-blobs-by-tags-container
```